draft-ietf-dnsop-svcb-https.md

```diff
@@ -715,10 +726,10 @@ the client MAY prefer those records, regardless of their priorities.
 ## Structuring zones for performance
 
 To avoid a delay for clients using a nonconforming recursive resolver,
-domain owners SHOULD use a single SVCB record whose SvcDomainName is
-"." if possible.  This will ensure that the required
-address records are already present in the client's DNS cache as part of the
-responses to the address queries that were issued in parallel.
+domain owners SHOULD minimize the use of AliasForm records, and choose
+SvcDomainName to be a domain for which the client will have already issued
+address queries (see {{client-behavior}}).  The simplest such configuration
+would be a single SVCB record whose SvcDomainName is the origin hostname.
```

Comment:
```suggestion
domain owners SHOULD minimize the use of AliasForm records, and choose
SvcDomainName to be a domain for which the client will have already issued
address queries (see {{client-behavior}}), such as by using "." as the SvcDomainName for ServiceForm records.  The simplest SVCB configuration
would be a single ServiceForm SVCB record whose SvcDomainName is the origin hostname.  (Note that using the origin hostname as the SvcDomainName can be problematic when using {{!ECH}} 
({{svcparamkeys-echconfig}}) and switching between different
endpoints as clients may get IP addresses for one endpoint
and {{svcparamkeys-echconfig}} for a different endpoint.)
```

I'm not sure if the note should be moved elsewhere, but it's an important warning for explaining why using the simplest form may not be preferable.

Reply:
Rather than describe a simple configuration and then advise against it, I changed this section to have an explicit example that we would actually recommend, using a layer of indirection as intended.